python/GangaLHCb/test/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM hepsw/cvmfs-lhcb:latest
+
+COPY ./ /root/ganga
+
+ENTRYPOINT . /etc/cvmfs/run-cvmfs.sh && \
+           . ~/.bashrc && \
+           export X509_CERT_DIR=/cvmfs/lhcb.cern.ch/etc/grid-security/certificates && \
+           export X509_VOMS_DIR=/cvmfs/lhcb.cern.ch/etc/grid-security/vomsdir && \
+           lhcb-proxy-init && \
```

Comment:
don't you have to source the group login from cvmfs before you can do this or is this done through .bashrc?